src/components/selector/selector.scss

```diff
@@ -0,0 +1,8 @@
+select {
```

Comment:
You are not supposed to style html tags. We are using BEM convention. Use classes instead.